sentry-spring-boot-starter/src/main/java/io/sentry/spring/boot/SentryAutoConfiguration.java

```diff
@@ -139,6 +140,19 @@ static class ContextTagsEventProcessorConfiguration {
       }
     }
 
+    @Configuration(proxyBeanMethods = false)
+    @ConditionalOnProperty(name = "sentry.auto-init", havingValue = "false")
```

Comment:
When not using the agent this will not trigger. At the moment it feels like a niche use case to not use the agent JAR and there doesn't seem to be an easy way of detecting whether the agent is in use. We could add some class that serves as a marker (to the bootstrap classloader) or manipulate the options (byte code manipulation) from the agent but decided not to do that for now. If this becomes a more common use case we can revisit.

Reply:
- 💡 could also look at the classloader of `OpenTelemetryLinkErrorEventProcessor`. If it's the bootstrap classloader the agent is in use. Not sure yet how to turn that into an `@Conditional` for spring config.

Reply:
- 💡 probably event easier to just have `SentryOptions` injected into the `@Bean` method and not create the bean if a `OpenTelemetryLinkErrorEventProcessor` has already been registered. Also not sure how to add that to the auto config yet.